pkg/crc/preflight/preflight_checks_windows.go

```diff
@@ -72,18 +91,15 @@ func checkHyperVInstalled() (bool, error) {
 	return true, nil
 }
 
-//
-func fixHyperVInstalled() (bool, error) {
-	enableHyperVCommand := `Enable-WindowsOptionalFeature -Online -FeatureName Microsoft-Hyper-V -All`
-	_, _, err := powershell.ExecuteAsAdmin("enable Hyper-V", enableHyperVCommand)
+func fixHyperVServiceRunning() (bool, error) {
+	enableVmmsService := `Set-Service -Name vmms -StartupType Automatic; Set-Service -Name vmms -Status Running -PassThru`
+	_, _, err := powershell.ExecuteAsAdmin("enable Hyper-V service", enableVmmsService)
 
 	if err != nil {
 		logging.Debug(err.Error())
-		return false, errors.New("Error occurred installing Hyper-V")
+		return false, errors.New("Error occurred enabling Hyper-V service")
 	}
 
-	// We do need to error out as a restart might be needed (unfortunately no output redirect possible)
-	logging.Error("Please reboot your system")
```

Comment:
No restart is needed after enabling the service?

Reply:
The services is enabled and started:

```
Set-Service -Name vmms -StartupType Automatic;
Set-Service -Name vmms -Status Running -PassThru
```